alarm/models.py

```diff
@@ -13,9 +16,57 @@ def __str__(self):
 
 
 class Coin(models.Model):
-    phone = models.ForeignKey(Phone, on_delete=models.CASCADE)
-    coin_abbreviation = models.CharField(max_length=255)
+    phone = models.ForeignKey(Phone, on_delete=models.CASCADE, default=None)
+    abbreviation = models.CharField(max_length=255, default=None)
```

Comment:
why we need default=None here? It's open possibility to errors

Reply:
++

Reply:
You are right. Deleted